src/modules/python/pbs/v1/_svr_types.py

```diff
@@ -256,7 +264,7 @@ class _job(object):
     _attributes_hook_set = {}
 
     def __new__(cls,value,connect_server=None):
-        return object.__new__(cls, value)
+        return object.__new__(cls)
```

Comment:
don't we have to pass `value` to `__new__()` ?

Reply:
new does not take any value. If we try to pass a value to new, it throws an error like this - 
in __new__\n    return object.__new__(cls, value)\n', 'TypeError: object() takes no parameters\n']